src/PdfReader/index.tsx

```diff
@@ -319,43 +401,63 @@ export default function usePdfReader(args: ReaderArguments): ReaderReturn {
     });
   }
 
+  function onRenderSuccess(page: PageProps) {
+    if (!page.height || !page.width || !containerSize) return;
```

Comment:
Should we maybe throw an error in this case? Otherwise they'll just have a loading screen forever right? Idk if this is a case that might actually happen or not tho